packages/paste-website/src/components/site-wrapper/sidebar/SidebarAnchor.tsx

```diff
@@ -1,86 +1,25 @@
 import * as React from 'react';
-import {styled, css} from '@twilio-paste/styling-library';
 import Link from 'next/link';
-import type {BoxStyleProps} from '@twilio-paste/box';
+import {SidebarNavigationItem} from '@twilio-paste/sidebar';
 
-import {SidebarItem} from './SidebarItem';
 import {useLocationPathname} from '../../../utils/RouteUtils';
 
 interface SidebarAnchorProps {
   children: NonNullable<React.ReactNode>;
-  level: 0 | 1 | 2;
   href: string;
   onClick?: () => void;
 }
 
-const levelPaddingMap: Partial<{
-  [key in SidebarAnchorProps['level']]: Partial<BoxStyleProps>;
-}> = {
-  0: {
-    paddingY: 'space40',
-    paddingLeft: ['space110', 'space110', 'space90'],
-    paddingRight: 'space40',
-  },
-  1: {
-    paddingY: 'space30',
-    paddingLeft: ['space130', 'space130', 'space110'],
-    paddingRight: 'space40',
-  },
-  2: {
-    paddingY: 'space30',
-    paddingLeft: ['space150', 'space150', 'space130'],
-    paddingRight: 'space40',
-  },
-};
-
-const StyledSidebarAnchor = styled(Link, {
-  shouldForwardProp: (prop) => prop !== 'level',
-})<SidebarAnchorProps>(({level}) =>
-  css({
-    position: 'relative',
-    display: 'block',
-    width: '100%',
-    fontWeight: 'fontWeightNormal',
-    color: 'colorText',
-    textDecoration: 'none',
-    transition: '0.1s background-color ease-in-out',
-    outline: 'none',
-    borderRadius: 'borderRadius10',
-    ...levelPaddingMap[level],
-
-    "&[aria-current='page']": {
-      backgroundColor: 'colorBackgroundStrong',
-      color: 'colorText',
-      fontWeight: 'fontWeightBold',
-    },
-
-    '&:hover': {
-      color: 'colorText',
-      textDecoration: 'underline',
-    },
-
-    '&:focus': {
-      boxShadow: 'shadowFocus',
-      textDecoration: 'underline',
-    },
-  })
-);
-
-const SidebarAnchor: React.FC<React.PropsWithChildren<SidebarAnchorProps>> = ({children, level, href, onClick}) => {
+const SidebarAnchor: React.FC<React.PropsWithChildren<SidebarAnchorProps>> = ({children, href, onClick}) => {
   const pathname = useLocationPathname();
   const pathnameWithoutTrailingSlash = pathname.endsWith('/') ? pathname.slice(0, -1) : pathname;
 
   return (
-    <SidebarItem>
-      <StyledSidebarAnchor
-        level={level}
-        href={href}
-        onClick={onClick}
-        aria-current={pathnameWithoutTrailingSlash === href ? 'page' : false}
-      >
+    <Link href={href} legacyBehavior>
+      <SidebarNavigationItem href={href} onClick={onClick} selected={pathnameWithoutTrailingSlash === href}>
```

Comment:
issue: We got two hrefs here, this is unexpected... should we pass `passHref` to the `Link` element? https://nextjs.org/docs/pages/api-reference/components/link#if-the-child-is-a-functional-component